05-20 14:27:33 jdbc[2]: exception
org.h2.jdbc.JdbcSQLException: Table "USERS" not found; SQL statement:
SELECT * FROM users WHERE username = 'username' [42102-170]
05-20 14:37:14 jdbc[2]: exception
org.h2.jdbc.JdbcSQLException: Table "USERS" not found; SQL statement:
DROP TABLE users [42102-170]
05-20 14:37:25 jdbc[2]: exception
org.h2.jdbc.JdbcSQLException: Table "USERS" not found; SQL statement:
DROP TABLE users [42102-170]
05-20 14:37:45 jdbc[2]: exception
org.h2.jdbc.JdbcSQLException: Table "FILES" not found; SQL statement:
DROP TABLE files [42102-170]
05-22 16:29:51 jdbc[2]: exception
org.h2.jdbc.JdbcSQLException: Invalid value "4" for parameter "parameterIndex" [90008-170]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:329)
	at org.h2.message.DbException.get(DbException.java:169)
	at org.h2.message.DbException.getInvalidValueException(DbException.java:215)
	at org.h2.jdbc.JdbcPreparedStatement.setParameter(JdbcPreparedStatement.java:1284)
	at org.h2.jdbc.JdbcPreparedStatement.setInt(JdbcPreparedStatement.java:321)
	at server.databases.H2Requests.addRequest(H2Requests.java:116)
	at server.databases.H2Requests.main(H2Requests.java:62)
05-23 15:11:40 jdbc[2]: exception
org.h2.jdbc.JdbcSQLException: Out of memory.; SQL statement:
INSERT INTO files VALUES (?,?,?,?) [90108-170]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:329)
	at org.h2.message.DbException.get(DbException.java:158)
	at org.h2.message.DbException.convert(DbException.java:275)
	at org.h2.command.Command.executeUpdate(Command.java:234)
	at org.h2.jdbc.JdbcPreparedStatement.executeUpdateInternal(JdbcPreparedStatement.java:156)
	at org.h2.jdbc.JdbcPreparedStatement.executeUpdate(JdbcPreparedStatement.java:142)
	at server.databases.H2Files.addFile(H2Files.java:124)
	at server.operations.ServerFileOperations.addFile(ServerFileOperations.java:25)
	at server.CentralAuthority.uploadEncryptedFile(CentralAuthority.java:95)
	at server.CentralAuthority.options(CentralAuthority.java:40)
	at server.Protocol.begin(Protocol.java:33)
	at server.ServerThread.run(ServerThread.java:23)
Caused by: java.lang.OutOfMemoryError: Java heap space
	at java.util.Arrays.copyOf(Arrays.java:2798)
	at java.io.ByteArrayOutputStream.write(ByteArrayOutputStream.java:111)
	at org.h2.util.IOUtils.copy(IOUtils.java:169)
	at org.h2.util.IOUtils.readBytesAndClose(IOUtils.java:277)
	at org.h2.value.ValueLobDb.getBytesNoCopy(ValueLobDb.java:232)
	at org.h2.value.Value.convertTo(Value.java:723)
	at org.h2.value.ValueLobDb.convertTo(ValueLobDb.java:120)
	at org.h2.table.Column.convert(Column.java:143)
	at org.h2.command.dml.Insert.insertRows(Insert.java:112)
	at org.h2.command.dml.Insert.update(Insert.java:84)
	at org.h2.command.CommandContainer.update(CommandContainer.java:75)
	at org.h2.command.Command.executeUpdate(Command.java:230)
	... 8 more
05-23 15:11:42 database: disconnecting session #2
org.h2.message.DbException: The database has been closed [90098-170]
	at org.h2.message.DbException.get(DbException.java:169)
	at org.h2.message.DbException.get(DbException.java:146)
	at org.h2.message.DbException.get(DbException.java:135)
	at org.h2.engine.Database.checkPowerOff(Database.java:432)
	at org.h2.engine.Session.close(Session.java:559)
	at org.h2.engine.Database.closeAllSessionsException(Database.java:1051)
	at org.h2.engine.Database.close(Database.java:1087)
	at org.h2.engine.DatabaseCloser.run(DatabaseCloser.java:80)
Caused by: org.h2.jdbc.JdbcSQLException: The database has been closed [90098-170]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:329)
	... 8 more
05-23 15:11:42 database: close
org.h2.message.DbException: The database has been closed; SQL statement:
SELECT ID FROM INFORMATION_SCHEMA.LOBS WHERE TABLE = ? [90098-170]
	at org.h2.message.DbException.convert(DbException.java:269)
	at org.h2.store.LobStorage.removeAllForTable(LobStorage.java:180)
	at org.h2.engine.Database.close(Database.java:1109)
	at org.h2.engine.DatabaseCloser.run(DatabaseCloser.java:80)
Caused by: org.h2.jdbc.JdbcSQLException: The database has been closed; SQL statement:
SELECT ID FROM INFORMATION_SCHEMA.LOBS WHERE TABLE = ? [90098-170]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:329)
	at org.h2.message.DbException.get(DbException.java:169)
	at org.h2.message.DbException.get(DbException.java:146)
	at org.h2.message.DbException.get(DbException.java:135)
	at org.h2.engine.Database.checkPowerOff(Database.java:432)
	at org.h2.command.Command.executeQuery(Command.java:189)
	at org.h2.jdbc.JdbcPreparedStatement.executeQuery(JdbcPreparedStatement.java:109)
	at org.h2.store.LobStorage.removeAllForTable(LobStorage.java:174)
	... 2 more
05-24 17:43:41 jdbc[2]: exception
org.h2.jdbc.JdbcSQLException: Table "KEYS" already exists; SQL statement:
CREATE TABLE keys(owner varchar(25) NOT NULL,securityLevel int(25) NOT NULL,key binary(256) NOT NULL,PRIMARY KEY (owner, securityLevel)) [42101-170]
05-24 18:31:43 jdbc[2]: exception
org.h2.jdbc.JdbcSQLException: Column count does not match; SQL statement:
INSERT INTO keys VALUES (?,?, ?, ?) [21002-170]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:329)
	at org.h2.message.DbException.get(DbException.java:169)
	at org.h2.message.DbException.get(DbException.java:146)
	at org.h2.message.DbException.get(DbException.java:135)
	at org.h2.command.dml.Insert.prepare(Insert.java:226)
	at org.h2.command.Parser.prepareCommand(Parser.java:218)
	at org.h2.engine.Session.prepareLocal(Session.java:414)
	at org.h2.engine.Session.prepareCommand(Session.java:363)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1116)
	at org.h2.jdbc.JdbcPreparedStatement.<init>(JdbcPreparedStatement.java:74)
	at org.h2.jdbc.JdbcConnection.prepareStatement(JdbcConnection.java:267)
	at server.databases.H2Keys.addKey(H2Keys.java:120)
	at server.operations.KeyOperations.storeKeys(KeyOperations.java:18)
	at server.operations.UserOperations.register(UserOperations.java:94)
	at server.operations.UserOperations.registerAttempt(UserOperations.java:63)
	at server.Protocol.begin(Protocol.java:29)
	at server.ServerThread.run(ServerThread.java:23)
05-24 18:32:53 jdbc[2]: exception
org.h2.jdbc.JdbcSQLException: Column count does not match; SQL statement:
INSERT INTO keys VALUES (?,?, ?, ?) [21002-170]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:329)
	at org.h2.message.DbException.get(DbException.java:169)
	at org.h2.message.DbException.get(DbException.java:146)
	at org.h2.message.DbException.get(DbException.java:135)
	at org.h2.command.dml.Insert.prepare(Insert.java:226)
	at org.h2.command.Parser.prepareCommand(Parser.java:218)
	at org.h2.engine.Session.prepareLocal(Session.java:414)
	at org.h2.engine.Session.prepareCommand(Session.java:363)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1116)
	at org.h2.jdbc.JdbcPreparedStatement.<init>(JdbcPreparedStatement.java:74)
	at org.h2.jdbc.JdbcConnection.prepareStatement(JdbcConnection.java:267)
	at server.databases.H2Keys.addKey(H2Keys.java:120)
	at server.operations.KeyOperations.storeKeys(KeyOperations.java:18)
	at server.operations.UserOperations.register(UserOperations.java:94)
	at server.operations.UserOperations.registerAttempt(UserOperations.java:63)
	at server.Protocol.begin(Protocol.java:29)
	at server.ServerThread.run(ServerThread.java:23)
05-24 18:32:53 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Column count does not match; SQL statement:
INSERT INTO keys VALUES (?,?, ?, ?) [21002-170]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:329)
	at org.h2.message.DbException.get(DbException.java:169)
	at org.h2.message.DbException.get(DbException.java:146)
	at org.h2.message.DbException.get(DbException.java:135)
	at org.h2.command.dml.Insert.prepare(Insert.java:226)
	at org.h2.command.Parser.prepareCommand(Parser.java:218)
	at org.h2.engine.Session.prepareLocal(Session.java:414)
	at org.h2.engine.Session.prepareCommand(Session.java:363)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1116)
	at org.h2.jdbc.JdbcPreparedStatement.<init>(JdbcPreparedStatement.java:74)
	at org.h2.jdbc.JdbcConnection.prepareStatement(JdbcConnection.java:267)
	at server.databases.H2Keys.addKey(H2Keys.java:120)
	at server.operations.KeyOperations.storeKeys(KeyOperations.java:18)
	at server.operations.UserOperations.register(UserOperations.java:94)
	at server.operations.UserOperations.registerAttempt(UserOperations.java:63)
	at server.Protocol.begin(Protocol.java:29)
	at server.ServerThread.run(ServerThread.java:23)
05-24 18:32:53 jdbc[4]: exception
org.h2.jdbc.JdbcSQLException: Column count does not match; SQL statement:
INSERT INTO keys VALUES (?,?, ?, ?) [21002-170]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:329)
	at org.h2.message.DbException.get(DbException.java:169)
	at org.h2.message.DbException.get(DbException.java:146)
	at org.h2.message.DbException.get(DbException.java:135)
	at org.h2.command.dml.Insert.prepare(Insert.java:226)
	at org.h2.command.Parser.prepareCommand(Parser.java:218)
	at org.h2.engine.Session.prepareLocal(Session.java:414)
	at org.h2.engine.Session.prepareCommand(Session.java:363)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1116)
	at org.h2.jdbc.JdbcPreparedStatement.<init>(JdbcPreparedStatement.java:74)
	at org.h2.jdbc.JdbcConnection.prepareStatement(JdbcConnection.java:267)
	at server.databases.H2Keys.addKey(H2Keys.java:120)
	at server.operations.KeyOperations.storeKeys(KeyOperations.java:18)
	at server.operations.UserOperations.register(UserOperations.java:94)
	at server.operations.UserOperations.registerAttempt(UserOperations.java:63)
	at server.Protocol.begin(Protocol.java:29)
	at server.ServerThread.run(ServerThread.java:23)
05-24 18:32:53 jdbc[5]: exception
org.h2.jdbc.JdbcSQLException: Column count does not match; SQL statement:
INSERT INTO keys VALUES (?,?, ?, ?) [21002-170]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:329)
	at org.h2.message.DbException.get(DbException.java:169)
	at org.h2.message.DbException.get(DbException.java:146)
	at org.h2.message.DbException.get(DbException.java:135)
	at org.h2.command.dml.Insert.prepare(Insert.java:226)
	at org.h2.command.Parser.prepareCommand(Parser.java:218)
	at org.h2.engine.Session.prepareLocal(Session.java:414)
	at org.h2.engine.Session.prepareCommand(Session.java:363)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1116)
	at org.h2.jdbc.JdbcPreparedStatement.<init>(JdbcPreparedStatement.java:74)
	at org.h2.jdbc.JdbcConnection.prepareStatement(JdbcConnection.java:267)
	at server.databases.H2Keys.addKey(H2Keys.java:120)
	at server.operations.KeyOperations.storeKeys(KeyOperations.java:18)
	at server.operations.UserOperations.register(UserOperations.java:94)
	at server.operations.UserOperations.registerAttempt(UserOperations.java:63)
	at server.Protocol.begin(Protocol.java:29)
	at server.ServerThread.run(ServerThread.java:23)
05-24 18:35:35 jdbc[2]: exception
org.h2.jdbc.JdbcSQLException: Column count does not match; SQL statement:
INSERT INTO keys VALUES (?,?,?,?) [21002-170]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:329)
	at org.h2.message.DbException.get(DbException.java:169)
	at org.h2.message.DbException.get(DbException.java:146)
	at org.h2.message.DbException.get(DbException.java:135)
	at org.h2.command.dml.Insert.prepare(Insert.java:226)
	at org.h2.command.Parser.prepareCommand(Parser.java:218)
	at org.h2.engine.Session.prepareLocal(Session.java:414)
	at org.h2.engine.Session.prepareCommand(Session.java:363)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1116)
	at org.h2.jdbc.JdbcPreparedStatement.<init>(JdbcPreparedStatement.java:74)
	at org.h2.jdbc.JdbcConnection.prepareStatement(JdbcConnection.java:267)
	at server.databases.H2Keys.addKey(H2Keys.java:120)
	at server.operations.KeyOperations.storeKeys(KeyOperations.java:18)
	at server.operations.UserOperations.register(UserOperations.java:94)
	at server.operations.UserOperations.registerAttempt(UserOperations.java:63)
	at server.Protocol.begin(Protocol.java:29)
	at server.ServerThread.run(ServerThread.java:23)
05-24 18:35:35 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Column count does not match; SQL statement:
INSERT INTO keys VALUES (?,?,?,?) [21002-170]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:329)
	at org.h2.message.DbException.get(DbException.java:169)
	at org.h2.message.DbException.get(DbException.java:146)
	at org.h2.message.DbException.get(DbException.java:135)
	at org.h2.command.dml.Insert.prepare(Insert.java:226)
	at org.h2.command.Parser.prepareCommand(Parser.java:218)
	at org.h2.engine.Session.prepareLocal(Session.java:414)
	at org.h2.engine.Session.prepareCommand(Session.java:363)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1116)
	at org.h2.jdbc.JdbcPreparedStatement.<init>(JdbcPreparedStatement.java:74)
	at org.h2.jdbc.JdbcConnection.prepareStatement(JdbcConnection.java:267)
	at server.databases.H2Keys.addKey(H2Keys.java:120)
	at server.operations.KeyOperations.storeKeys(KeyOperations.java:18)
	at server.operations.UserOperations.register(UserOperations.java:94)
	at server.operations.UserOperations.registerAttempt(UserOperations.java:63)
	at server.Protocol.begin(Protocol.java:29)
	at server.ServerThread.run(ServerThread.java:23)
05-24 18:35:35 jdbc[4]: exception
org.h2.jdbc.JdbcSQLException: Column count does not match; SQL statement:
INSERT INTO keys VALUES (?,?,?,?) [21002-170]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:329)
	at org.h2.message.DbException.get(DbException.java:169)
	at org.h2.message.DbException.get(DbException.java:146)
	at org.h2.message.DbException.get(DbException.java:135)
	at org.h2.command.dml.Insert.prepare(Insert.java:226)
	at org.h2.command.Parser.prepareCommand(Parser.java:218)
	at org.h2.engine.Session.prepareLocal(Session.java:414)
	at org.h2.engine.Session.prepareCommand(Session.java:363)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1116)
	at org.h2.jdbc.JdbcPreparedStatement.<init>(JdbcPreparedStatement.java:74)
	at org.h2.jdbc.JdbcConnection.prepareStatement(JdbcConnection.java:267)
	at server.databases.H2Keys.addKey(H2Keys.java:120)
	at server.operations.KeyOperations.storeKeys(KeyOperations.java:18)
	at server.operations.UserOperations.register(UserOperations.java:94)
	at server.operations.UserOperations.registerAttempt(UserOperations.java:63)
	at server.Protocol.begin(Protocol.java:29)
	at server.ServerThread.run(ServerThread.java:23)
05-24 18:35:35 jdbc[5]: exception
org.h2.jdbc.JdbcSQLException: Column count does not match; SQL statement:
INSERT INTO keys VALUES (?,?,?,?) [21002-170]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:329)
	at org.h2.message.DbException.get(DbException.java:169)
	at org.h2.message.DbException.get(DbException.java:146)
	at org.h2.message.DbException.get(DbException.java:135)
	at org.h2.command.dml.Insert.prepare(Insert.java:226)
	at org.h2.command.Parser.prepareCommand(Parser.java:218)
	at org.h2.engine.Session.prepareLocal(Session.java:414)
	at org.h2.engine.Session.prepareCommand(Session.java:363)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1116)
	at org.h2.jdbc.JdbcPreparedStatement.<init>(JdbcPreparedStatement.java:74)
	at org.h2.jdbc.JdbcConnection.prepareStatement(JdbcConnection.java:267)
	at server.databases.H2Keys.addKey(H2Keys.java:120)
	at server.operations.KeyOperations.storeKeys(KeyOperations.java:18)
	at server.operations.UserOperations.register(UserOperations.java:94)
	at server.operations.UserOperations.registerAttempt(UserOperations.java:63)
	at server.Protocol.begin(Protocol.java:29)
	at server.ServerThread.run(ServerThread.java:23)
05-27 23:37:39 jdbc[2]: exception
org.h2.jdbc.JdbcSQLException: Column count does not match; SQL statement:
INSERT INTO users VALUES (?,?,?,?,?,?,?) [21002-170]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:329)
	at org.h2.message.DbException.get(DbException.java:169)
	at org.h2.message.DbException.get(DbException.java:146)
	at org.h2.message.DbException.get(DbException.java:135)
	at org.h2.command.dml.Insert.prepare(Insert.java:226)
	at org.h2.command.Parser.prepareCommand(Parser.java:218)
	at org.h2.engine.Session.prepareLocal(Session.java:414)
	at org.h2.engine.Session.prepareCommand(Session.java:363)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1116)
	at org.h2.jdbc.JdbcPreparedStatement.<init>(JdbcPreparedStatement.java:74)
	at org.h2.jdbc.JdbcConnection.prepareStatement(JdbcConnection.java:267)
	at server.databases.H2Users.addUser(H2Users.java:166)
	at server.users.User.save(User.java:37)
	at server.users.Authentication.createUser(Authentication.java:76)
	at server.operations.UserOperations.register(UserOperations.java:105)
	at server.operations.UserOperations.registerAttempt(UserOperations.java:71)
	at server.Protocol.begin(Protocol.java:29)
	at server.ServerThread.run(ServerThread.java:23)
05-29 17:09:17 jdbc[2]: exception
org.h2.jdbc.JdbcSQLException: Data conversion error converting "9d4f4fad6a0aeec3f20732c8440c9cc2e4868186cb5807315c63e30d829e8e7dafeb241cf0f4a7685d2bc30129e8d775"; SQL statement:
UPDATE keys SET key= ? AND iv= ? WHERE owner = 'username' AND securityLevel = '2' [22018-170]
05-29 17:09:17 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Data conversion error converting "b7e1f88d6219afcd2c2d2b93798086cdf35c5356bd2c1e3d149cb5995eb1cbdf222aea5e1be34061ccd5d3495c45a371"; SQL statement:
UPDATE keys SET key= ? AND iv= ? WHERE owner = 'username' AND securityLevel = '3' [22018-170]
05-29 17:09:17 jdbc[4]: exception
org.h2.jdbc.JdbcSQLException: Data conversion error converting "ca4d7f0a310b5e01416e3576ab534073c2e617ee6102ac0c035b46f566d56a37c86ed9ff3d6a53373884bce373cedce6"; SQL statement:
UPDATE keys SET key= ? AND iv= ? WHERE owner = 'username' AND securityLevel = '4' [22018-170]
05-29 17:36:22 jdbc[2]: exception
org.h2.jdbc.JdbcSQLException: Data conversion error converting "802a47612e7c46fa2540fc6ff76b68c5cb483b3672bfacf11fc19271da53f0e1059dfd5b2aa9ad92c81a74412914d3f8"; SQL statement:
UPDATE keys SET key= ? AND iv= ? WHERE owner = 'username' AND securityLevel = 2 [22018-170]
05-29 17:36:22 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Data conversion error converting "2ae934f3b70b5cad77048e1cb8dde7933af91c48241bd0db2bbfecc3534d0f58ba07a200ab150d3f1d42b09a72f6430e"; SQL statement:
UPDATE keys SET key= ? AND iv= ? WHERE owner = 'username' AND securityLevel = 3 [22018-170]
05-29 17:36:22 jdbc[4]: exception
org.h2.jdbc.JdbcSQLException: Data conversion error converting "a5e961708b5b7212e47d24bf74f708b7957940bbd2dca015aab0786038243684c8b6535fa920b8e7555e811b069a5a3a"; SQL statement:
UPDATE keys SET key= ? AND iv= ? WHERE owner = 'username' AND securityLevel = 4 [22018-170]
06-03 13:08:14 jdbc[2]: exception
org.h2.jdbc.JdbcSQLException: Data conversion error converting "98f7db8c1dc8a7e6108d5b6c2a8d7e3894a15c825a06d54ebbbe4f499c662a238898ebffdb2f16f32a9cb8f540e456f4"; SQL statement:
UPDATE keys SET key= ? AND iv= ? WHERE owner = 'username' AND securityLevel = 2 [22018-170]
06-03 13:08:14 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Data conversion error converting "4d515f3e06c087fd72caf151d36ec3aadccfb796939960fee04818b7190777bed706ac318d5f7f1a489d2b49acdf442f"; SQL statement:
UPDATE keys SET key= ? AND iv= ? WHERE owner = 'username' AND securityLevel = 3 [22018-170]
06-03 13:08:14 jdbc[4]: exception
org.h2.jdbc.JdbcSQLException: Data conversion error converting "26125fdefe6be5c6f35ff81192f042f32b57e169ac5a5a146d0d053bb89737e4224206924e3a4177f0df0b6315c11ba7"; SQL statement:
UPDATE keys SET key= ? AND iv= ? WHERE owner = 'username' AND securityLevel = 4 [22018-170]
06-03 13:08:14 jdbc[5]: exception
org.h2.jdbc.JdbcSQLException: Data conversion error converting "018ddfd09cce2703b25eb9f78d81eff5d4ed1c0d5946f8616526cbf3203af8badf8fae3e5ce733684dcddf078c91e4bf"; SQL statement:
UPDATE keys SET key= ? AND iv= ? WHERE owner = 'username' AND securityLevel = 5 [22018-170]
06-03 13:45:10 jdbc[2]: exception
org.h2.jdbc.JdbcSQLException: Hexadecimal string with odd number of characters: "FALSE"; SQL statement:
UPDATE keys SET key= ? AND iv= ? WHERE owner = 'owner' AND securityLevel = 1 [90003-170]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:329)
	at org.h2.message.DbException.get(DbException.java:169)
	at org.h2.message.DbException.get(DbException.java:146)
	at org.h2.util.StringUtils.convertHexToBytes(StringUtils.java:990)
	at org.h2.value.Value.convertTo(Value.java:821)
	at org.h2.table.Column.convert(Column.java:143)
	at org.h2.command.dml.Update.update(Update.java:119)
	at org.h2.command.CommandContainer.update(CommandContainer.java:75)
	at org.h2.command.Command.executeUpdate(Command.java:230)
	at org.h2.jdbc.JdbcPreparedStatement.executeUpdateInternal(JdbcPreparedStatement.java:156)
	at org.h2.jdbc.JdbcPreparedStatement.executeUpdate(JdbcPreparedStatement.java:142)
	at server.databases.H2Keys.updateKey(H2Keys.java:170)
	at server.databases.H2Keys.main(H2Keys.java:79)
06-03 13:45:53 jdbc[2]: exception
org.h2.jdbc.JdbcSQLException: Hexadecimal string with odd number of characters: "FALSE"; SQL statement:
UPDATE keys SET key= ? AND iv= ? WHERE owner = 'owner' AND securityLevel = 1 [90003-170]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:329)
	at org.h2.message.DbException.get(DbException.java:169)
	at org.h2.message.DbException.get(DbException.java:146)
	at org.h2.util.StringUtils.convertHexToBytes(StringUtils.java:990)
	at org.h2.value.Value.convertTo(Value.java:821)
	at org.h2.table.Column.convert(Column.java:143)
	at org.h2.command.dml.Update.update(Update.java:119)
	at org.h2.command.CommandContainer.update(CommandContainer.java:75)
	at org.h2.command.Command.executeUpdate(Command.java:230)
	at org.h2.jdbc.JdbcPreparedStatement.executeUpdateInternal(JdbcPreparedStatement.java:156)
	at org.h2.jdbc.JdbcPreparedStatement.executeUpdate(JdbcPreparedStatement.java:142)
	at server.databases.H2Keys.updateKey(H2Keys.java:171)
	at server.databases.H2Keys.main(H2Keys.java:80)
06-03 13:49:44 jdbc[2]: exception
org.h2.jdbc.JdbcSQLException: Invalid value "3" for parameter "parameterIndex" [90008-170]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:329)
	at org.h2.message.DbException.get(DbException.java:169)
	at org.h2.message.DbException.getInvalidValueException(DbException.java:215)
	at org.h2.jdbc.JdbcPreparedStatement.setParameter(JdbcPreparedStatement.java:1284)
	at org.h2.jdbc.JdbcPreparedStatement.setBinaryStream(JdbcPreparedStatement.java:869)
	at org.h2.jdbc.JdbcPreparedStatement.setBinaryStream(JdbcPreparedStatement.java:889)
	at server.databases.H2Keys.updateKey(H2Keys.java:168)
	at server.databases.H2Keys.main(H2Keys.java:80)
06-03 13:49:53 jdbc[2]: exception
org.h2.jdbc.JdbcSQLException: Hexadecimal string with odd number of characters: "FALSE"; SQL statement:
UPDATE keys SET key= ? AND iv= ? WHERE owner = 'owner' AND securityLevel = 1 [90003-170]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:329)
	at org.h2.message.DbException.get(DbException.java:169)
	at org.h2.message.DbException.get(DbException.java:146)
	at org.h2.util.StringUtils.convertHexToBytes(StringUtils.java:990)
	at org.h2.value.Value.convertTo(Value.java:821)
	at org.h2.table.Column.convert(Column.java:143)
	at org.h2.command.dml.Update.update(Update.java:119)
	at org.h2.command.CommandContainer.update(CommandContainer.java:75)
	at org.h2.command.Command.executeUpdate(Command.java:230)
	at org.h2.jdbc.JdbcPreparedStatement.executeUpdateInternal(JdbcPreparedStatement.java:156)
	at org.h2.jdbc.JdbcPreparedStatement.executeUpdate(JdbcPreparedStatement.java:142)
	at server.databases.H2Keys.updateKey(H2Keys.java:171)
	at server.databases.H2Keys.main(H2Keys.java:80)
06-03 13:50:18 jdbc[2]: exception
org.h2.jdbc.JdbcSQLException: Hexadecimal string with odd number of characters: "FALSE"; SQL statement:
UPDATE keys SET key= ? AND iv= ? WHERE owner = 'owner' AND securityLevel = 1 [90003-170]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:329)
	at org.h2.message.DbException.get(DbException.java:169)
	at org.h2.message.DbException.get(DbException.java:146)
	at org.h2.util.StringUtils.convertHexToBytes(StringUtils.java:990)
	at org.h2.value.Value.convertTo(Value.java:821)
	at org.h2.table.Column.convert(Column.java:143)
	at org.h2.command.dml.Update.update(Update.java:119)
	at org.h2.command.CommandContainer.update(CommandContainer.java:75)
	at org.h2.command.Command.executeUpdate(Command.java:230)
	at org.h2.jdbc.JdbcPreparedStatement.executeUpdateInternal(JdbcPreparedStatement.java:156)
	at org.h2.jdbc.JdbcPreparedStatement.executeUpdate(JdbcPreparedStatement.java:142)
	at server.databases.H2Keys.updateKey(H2Keys.java:166)
	at server.databases.H2Keys.main(H2Keys.java:79)
06-03 13:51:01 jdbc[2]: exception
org.h2.jdbc.JdbcSQLException: Data conversion error converting "b7b631a6b2c5b4f291a1663a53486c909dc7bbcf1dfbc4fe394bc1d21ef2f7bc"; SQL statement:
UPDATE keys SET key= ? AND iv= ? WHERE owner = 'owner' AND securityLevel = 1 [22018-170]
06-03 14:01:27 jdbc[2]: exception
org.h2.jdbc.JdbcSQLException: Data conversion error converting "c2e1c2d90acc928715ba0d39a6725ddd5a6165e980c10912c9bc1b1ef2a2a7e3"; SQL statement:
UPDATE keys SET key=? AND iv=? WHERE owner = 'owner' AND securityLevel = 1 [22018-170]
06-03 15:36:09 jdbc[2]: exception
org.h2.jdbc.JdbcSQLException: Syntax error in SQL statement "SELECT * FROM FILES WHERE OWNER = 'username' ,[*] SECURITYLEVEL >= 3 "; SQL statement:
SELECT * FROM files WHERE owner = 'username' , securityLevel >= 3 [42000-170]
06-03 15:37:43 jdbc[2]: exception
org.h2.jdbc.JdbcSQLException: Syntax error in SQL statement "SELECT * FROM files WHERE owner = [*]'username"; SQL statement:
SELECT * FROM files WHERE owner = 'username [42000-170]
06-03 15:40:16 jdbc[2]: exception
org.h2.jdbc.JdbcSQLException: Syntax error in SQL statement "SELECT * FROM files WHERE owner = [*]'username "; SQL statement:
SELECT * FROM files WHERE owner = 'username  [42000-170]
